ecf/scripts/plots/cam/jevs_cam_href_spcoutlook_last31days_plots.ecf

```diff
@@ -1,10 +1,10 @@
-#PBS -N evs_cam_href_spcoutlook_past31days_plots
+#PBS -N evs_cam_href_spcoutlook_last31days_plots
 #PBS -j oe
 #PBS -S /bin/bash
 #PBS -q %QUEUE%
 #PBS -A %PROJ%-%PROJENVIR%
-#PBS -l walltime=00:10:00
-#PBS -l place=vscatter:shared,select=1:ncpus=12:mem=100GB
+#PBS -l walltime=00:15:00
+#PBS -l place=vscatter:shared,select=1:ncpus=5:mem=5GB
```

Comment:
Remove `:shared`, dev driver says `ncpus=6` but this says `ncpus=6`. Please fix in whichever script is wrong.

Reply:
Never mind I see the related changes now